Python/Python-tokenize.c

```diff
@@ -210,7 +222,20 @@ tokenizeriter_next(tokenizeriterobject *it)
         if (size >= 1 && it->tok->implicit_newline) {
             size -= 1;
         }
-        line = PyUnicode_DecodeUTF8(line_start, size, "replace");
+
+        if (line_start != it->last_line_start || size != it->last_line_size) {
```

Comment:
Under what circumstances would `line_start == it->last_line_start` but `size != it->last_line_size`?